src/app/pages/questions/components/question/audio-input/audio-input.component.ts

```diff
@@ -37,6 +37,7 @@ export class AudioInputComponent implements OnDestroy, OnInit {
   resumeListener: Subscription
   pauseListener: Subscription
   audioData: string
+  interruptedAlertShown: boolean
```

Comment:
maybe you should declare it with default value or also check for defined in the following check? I don't know whether typescript does the defined check automatically in this case

Reply:
I have initialized it to false.